rubocop/rubocop.yml

```diff
@@ -1,22 +1,45 @@
-Style/StringLiterals:
-  EnforcedStyle: double_quotes
+Layout/AlignParameters:
+  EnforcedStyle: with_fixed_indentation
+
+Layout/MultilineMethodCallIndentation:
+  EnforcedStyle: indented
+
+Metrics/BlockLength:
   Exclude:
-    - 'Gemfile'
+    - 'spec/**/*'
 
 Metrics/LineLength:
   Max: 100
   Severity: refactor
 
+Metrics/ModuleLength:
+  Exclude:
+    - 'spec/**/*'
```

Comment:
Might also want to exclude the `test` directory files as well (although that can be done in a separate PR.

Reply:
I intentionally want to keep the scope to this limited to the alphabetization, but happy to propose other modifications to cops separately.